sphinx/themes/basic/static/basic.css_t

```diff
@@ -311,6 +311,19 @@ img.align-default, figure.align-default, .figure.align-default {
     text-align: right;
 }
 
+#sphinx-skip-link {
+  position: absolute;
+  top: -1000%;
+  left: -1000%;
+  background-color: #fda;
+  padding: 1em;
+}
```

Comment:
new node will render like this. I used the same background that `system-message` uses. LMK if the background should be something else.

![image](https://github.com/sphinx-doc/sphinx/assets/4713486/885907f2-61cb-405c-8c01-9a36a2c9f6c2)